haystack/utils/squad_data.py

```diff
@@ -109,8 +109,13 @@ def to_df(data):
         """Convert a list of SQuAD document dictionaries into a pandas dataframe (each row is one annotation)"""
         flat = []
         for document in data:
-            title = document["title"]
+            title = ""
+            if "title" in document:
+                title = document["title"]
```

Comment:
A couple of style remark here:
- Using a dictionary as an iterable implicitly is generally not recommended (might be even raising an issue in pylint). You should always iterate explicitly either on `.keys()`, `.values()` or `.items()`. In this case you should iterate on `document.keys()`
- There's no need to check for the existence of a key before getting it. You can replace these three lines with:
```python
title = document.get("title", "")
```
A similar approach should work for all the other values you're retrieving after checking for the existence of the key.

Reply:
Yes that's much cleaner!